index.html

```diff
@@ -11,7 +11,7 @@
           shortName:  "ltli",
           editors: [
                 {   name:       "Addison Phillips",
-                    company:    "Invited Expert"
+                    company:    "Amazon.com"
 	    },
 	              {   name: "Felix Sasaki",
 		                company: "W3C"
```

Comment:
Can you change my affiliation to "invited expert"?

Reply:
Done.